data_relay.py

```diff
@@ -51,82 +47,158 @@
-            ('h', 'ch5out'),
-            ('h', 'ch6out'),
-            ('h', 'ch7out'),
-            ('h', 'ch8out'),
-            ('h', 'camera_status'),
+        PACKET_TYPE_STATUS: ( # 44 bytes + 36 bytes padding
+            ('h', 'roll_rate_setpoint'),
+            ('h', 'pitch_rate_setpoint'),
             ('h', 'yaw_rate_setpoint'),
+            ('h', 'roll_setpoint'),
+            ('h', 'pitch_setpoint'),
             ('h', 'heading_setpoint'),
             ('h', 'altitude_setpoint'),
-            ('h', 'flap_setpoint'),
-            ('B', 'wireless_connection'),
-            ('B', 'autopilot_active'),
-            ('B', 'gps_status'),
-            ('B', 'waypoint_count'),
+            ('h', 'throttle_setpoint'),
+            ('h', 'internal_battery_voltage'),
+            ('h', 'external_battery_voltage'),
+            ('H', 'program_state'),
+            ('H', 'autonomous_level'),
+            ('H', 'startup_errors'),
+            ('H', 'am_interchip_errors'),
+            ('H', 'pm_interchip_errors'),
+            ('H', 'gps_communication_errors'),
+            ('H', 'dl_transmission_errors'),
+            ('H', 'ul_receive_errors'),
+            ('H', 'peripheral_status'),
+            ('H', 'uhf_channel_status'),
+            ('B', 'ul_rssi'),
+            ('B', 'uhf_rssi'),
+            ('B', 'uhf_link_quality'),
             ('B', 'waypoint_index'),
-            ('B', 'path_following'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding')
```

Comment:
This packet has ridiculous amounts of padding....Sorta in efficient. I understand what you are trying to do. It comes at a cost however: If you are wasting all this data, might as well include "Waypoint Count" and "Path Following" <true/false value>. You seemed to have gotten rid of them.